Sources/StreamChat/APIClient/AttachmentDownloader/AttachmentDownloader.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The component responsible for downloading files.
+public protocol AttachmentDownloader {
```

Comment:
Should this be public? I think we can make it internal for now. If some customer wants to provide their own downloader for some reason, we can then make it public in the future 👍

Reply:
Removed